frontend/util/stringUtils.ts

```diff
@@ -0,0 +1,9 @@
+export function hasTrailingChar(text: string): boolean {
+  return (text.endsWith(',') || text.endsWith('.'));
```

Comment:
Should probably also match to other invalid chars like !, more dificult would be the ? as that is also valid in urls to start a query argument

Reply:
Added `!` and `?`. Who *ends* an URL with `?` anyway? .shrug